pom.xml

```diff
@@ -42,7 +42,7 @@
   </organization>
 
   <properties>
-    <hadoop.version>2.6.0-cdh5.5.0</hadoop.version>
+    <hadoop.version>2.6.5</hadoop.version>
```

Comment:
Yes, personally I don't have a strong preference on which version of Hadoop to use, I just keep align with Spark's minimum supported version of Hadoop.